dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -133,10 +146,298 @@ xref:../admin_guide/high_availability.adoc#admin-guide-high-availability[VIP] ad
 - In a non-cloud environment, by configuring ingressIP range (`ingressIPNetworkCIDR`),
 `service.type=LoadBalancer`, and `service.port.ingressIP`.
 
-The administrator must ensure the external IPs are routed to the nodes and local firewall rules on all nodes allow access to the open port.
+The administrator must ensure the external IPs are routed to the nodes and local
+firewall rules on all nodes allow access to the open port.
+
+Use the same nodes as the router, but advertise them with VRRP and use the DNS
+address of the router nodes.
+
+.Example External IP Configured within a Service Definition
+----
+$ oc create -f - <<INGRESS
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgresql-ingress
+spec:
+  externalIPs:
+  - 10.9.54.100 <1>
+  ports:
+  - port: 5432
+    protocol: TCP
+  selector:
+    name: postgresql
+  type: LoadBalancer
+INGRESS
+----
+<1> The IP address assigned to the external IP.
+
+The external IP address is not managed by the underlying Kubernetes
+infrastructure and must be maintained and provided by a cluster administrator.
+While external IPs provide a solution for accessing services on the {product-tlte}
+cluster, there are several shortcomings:
+
+* Application users cannot create external IP addresses themselves, and must work
+with cluster administrators to allocate an address for their usage.
+* There are no protections in place to restrict the usage of a particular external
+address configured within the cluster. This allows the potential for a single
+address to be used by multiple services targeting the same port. When this
+situation occurs, the service which requested the port first is given use of the
+port.
+
+xref:using-ingress[Ingress] provides a solution.
+
+[[using-ingress]]
+=== Ingress
+
+Ingress streamlines the allocation of External IPs for accessing to services in
+the cluster. Cluster administrators can designate a range of addresses using a
+CIDR notation, which allows an application user to make a request against the
+cluster for an external IP address. When a service is configured with the type
+`LoadBalancer`, an External IP address will be automatically assigned from the
+designated range.
+
+[NOTE]
+====
+Ingress is only applicable in non-cloud based environments.
+====
+
+A common use case for Ingress is the ability to provide database services, such
+as PostgreSQL, to clients outside of the {product-title} cluster.
+
+[[using-ingress-defining-the-ingress-IP-range]]
+==== Defining the Ingress IP Range
+
+The ability for cluster administrators to automatically allocate External IP
+addresses using Ingress is enabled by default within {product-title} and
+configured to use the `172.46.0.0/16` range. An alternate range can be specified
+by configuring the `ingressIPNetworkCIDR` parameter in the
+*_/etc/origin/master-config.yaml_* file:
+
```

Comment:
If an IP address was allocated from the ingressIPNetworkCIDR range, and then the range is changed I believe the service's externalIP gets reallocated an address that is in the new range.  @marun is that correct?

Maru also mentioned that the IP allocations _could_ change at any time, but likely would not... Maru, am I misrepresenting your statement, and if not, can you clarify what we want to say in the docs?